R/build-home.R

```diff
@@ -269,6 +270,29 @@ on_cran <- function(pkg, cran = cran_mirror()) {
   pkg %in% rownames(pkgs)
 }
 
+on_bioc <- function(pkg) {
+  contrib <-
+    if (requireNamespace("BiocInstaller")) {
```

Comment:
Can you please put in the assignment inside the `if` statement?

And you need `quietly = TRUE` in `requireNamespace()`